src/index.js

```diff
@@ -1,28 +1,16 @@
 import './style.css';
 // eslint-disable-next-line import/no-cycle
```

Comment:
- [OPTIONAL]  `Linters` are tools that perform static analysis on software in order to recognize and report adherence/non-adherence to some set of coding best practice.Instead of disabling `eslint`, it will be nice to fix the issue.